release/nightly_tests/dataset/tpch_q1.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+from datetime import datetime, timedelta
+from typing import Dict
+
+import numpy as np
+import ray
+
+# TODO: We should make these public again.
+from ray.data._internal.aggregate import Count, Sum, Mean
+
+from benchmark import Benchmark
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description="TPCH Q1")
+    parser.add_argument("--sf", choices=[1, 10, 100, 1000, 10000], type=int, default=1)
+    return parser.parse_args()
+
+
+def main(args):
+    path = f"s3://ray-benchmark-data/tpch/parquet/sf{args.sf}/lineitem"
+    benchmark = Benchmark("tpch-q1")
+
```

Comment:
Nice!

To work around the column names annoyance, we can add rename_columns after read_parquet or just use a Dict of the current Dataset column names to TPCH Lineitem column names. This should improve the code readability significantly.